servicebox-app/src/test/java/io/halkyon/ApplicationsPageTest.java

```diff
@@ -157,16 +157,16 @@ public void testBindApplication(){
         assertNotNull(actualClaim.url);
         assertEquals("testbind://" + serviceName + ":1111",actualClaim.url);
 
-
         // then secret should have been generated
         String url = serviceName + ":1111";
         String urlBase64 = Base64.getEncoder().encodeToString(url.getBytes());
         String userBase64 = Base64.getEncoder().encodeToString("user1".getBytes());
         String pwdBase64 = Base64.getEncoder().encodeToString("pass1".getBytes());
+        String typeBase64 = Base64.getEncoder().encodeToString("type1".getBytes());
         verify(mockKubernetesClientService, times(1))
                 .mountSecretInApplication(argThat(new ApplicationNameMatcher(appName)),
```

Comment:
You need to add the new param `type`to the [KubernetesClientService#mountSecret method](https://github.com/halkyonio/primaza-poc/blob/d69362960eb59f1f31ef7e91e5db229223657a15/servicebox-app/src/main/java/io/halkyon/services/KubernetesClientService.java#L122)

Reply:
Fixed. Many thanks